Landingspagina/Index.html

```diff
@@ -0,0 +1,238 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="styles.css">
+    <title>Landingspagina</title>
+</head>
+
+<body>
+
+<header class="outer-container">
+
+    <div class="inner-container">
+
+        <nav>
+            <ul class="Menu">
+                <li>Case Studies</li>
+                <li>Services</li>
+                <li>Contact Us</li>
+            </ul>
+        </nav>
+
+
+        <div class="header-content content-restrictor">
+            <h1>Your website is an investment in the success of your business.</h1>
+            <p id="p-slogan">We are here to help you succeed.</p>
+
+            <a href="#work-together" class="button">Let's work together</a>
+        </div>
+
+        <img class="header-im" src="Assets/squiggle_line_stripe.svg"/>
+    </div>
+
+</header>
+
+
+<main>
+    <section class="outer-container">
+
+        <div class="inner-container content-restrictor d-2">
+            <h2 class="h2">Rekupe is a digital agency in Los Angeles.</h2>
+            <h2>We use strategic UX, SEO optimization and beautiful visual designs to help your business grow.</h2>
+            <h2>We see every project as a fresh opportunity, pushing ourselves to create new digital experiences.</h2>
+            <p class="klik-tekst">Have a question? <span style="color:#5722CD;font-weight:bold">Click here </span> to
+                contact us!</p>
+        </div>
+    </section>
+
+    <section class="outer-container grey">
+
+        <div class="inner-container">
+
+            <article class="project-article">
+                <div class="d-article">
+
+                    <h2 class="h-article">Rolling Stone</h2>
+
+                    <span class="image-wrapper purple-squiggle">
+                        <img class="purple-squiggle-im" src="Assets/squiggle_line.svg" alt="purple-squiggle"/>
```

Comment:
Voor alle afbeeldingen geldt dat de assets niet in de repository staan waardoor ik de afbeeldingen niet kan zien

Reply:
Ohja klopt ik heb het in gitignore gezet. Aanvankelijk waren ze toch wel meegekomen, maar ik moest uiteindelijk een nieuwe repository maken omdat ik de opdracht onbedoeld in de main had gezet en er dus geen comments gezet zouden kunnen worden. De tweede keer zij de assets wel genegeerd. Ik heb ze net toegevoegd.